packages/cli/internal/pkg/cli/context/manager.go

```diff
@@ -87,8 +87,16 @@ var showExecution = cdk.ShowExecution
 var silentExecution = cdk.SilentExecution
 
 func (m *Manager) getEnvironmentVars() []string {
+	// Different engines will need different environment variables to define
+	// their Docker images.
+	engine := m.contextEnv.EngineName
+	// Each engine has its own section in imageRefs, and for now we assume they
+	// all care about a WES adapter image.
```

Comment:
I don't think we should assume this. Toils is an example of an engine with no WES Adapter image so rather than faking it we should handle the edge case where there is no WES adapter

Reply:
I can make it support passing the WES adapter for the engines that need it only.